serverless/src/forwarder.ts

```diff
@@ -55,8 +56,10 @@ export class MissingSubDeclarationError extends Error {
  * log group for each lambda function.
  *
  * We first check if a log group exists for a given lambda. If it does, we then check if there
- * are existing subscriptions. We can go ahead and add the subscription to the forwarder ARN
- * (using AWS SDK) if no other unknown subscriptions exist.
+ * are existing subscriptions. We cannot add the subscription to the forwarder ARN if the number
+ * of existing subscriptions equals the maximum number allowed by AWS cloudwatch logs.
+ * We can go ahead and add the subscription to the forwarder ARN
+ * (using AWS SDK) in any other case.
```

Comment:
What about the case where there is only one subscription, but it is ours? Looks like this comment doesn't match the logic in the code

Reply:
I changed that comment to 
* We first check if a log group exists for a given lambda. If it does, we then check if there
 * are existing subscriptions. If we do not find an existing datadog-cloudformation-macro subscription and
 * the number of existing subscription filters is less than the MAX_ALLOWABLE_LOG_GROUP_SUBSCRIPTIONS then 
 * we will go ahead and add the subscription to the forwarder ARN (using AWS SDK). Otherwise we do not add 
 * the subscription.